common/scripts/base.sh

```diff
@@ -99,8 +99,7 @@ function undeploy() {
 # $2 - service port
 function waitUntilServiceIsActive() {
   marathon_lb_host_ip=$(dcos marathon app show marathon-lb | $mesos_artifacts_home/common/scripts/get-marathon-lb-host.py)
```

Comment:
@Thanu we are only checking service ports in marathon-lb (host machine in which marathon-lb is created). But for mysql dbs, we need to get the host machine IPs of relevant mysql apps and check the host ports in those IPs.

@isurulucky we can't use marathon-lb's hostname since mysql dbs are directly accessed without marathon-lb